README.md

```diff
@@ -34,4 +34,6 @@ console.log(a)
 ```
 
 
-## License [MIT](LICENSE)
```

Comment:
I thought that you paste license name without link to file (but checked you other repos and I was wrong)